src/open_inwoner/accounts/models.py

```diff
@@ -367,7 +360,7 @@ def get_short_name(self):
 
     @property
     def call_name(self):
-        return self.display_name or self.first_name
+        return self.first_name or self.first_name
```

Comment:
"return a or a" is redundant. Context: I introduced `call_name` to show either the `display_name` or the `first_name`, in case the `display_name` was not set. Now that we're getting rid of the model field `display_name`, I suggest renaming the propery to "display_name" and return the first name for now. If some municipalities decide they want to display user names in a different way, we can easily customize it:

```
@property
def display_name(self)
    return self.first_name
```
You'll need to change it in `profile/me.html` as well